src/App.jsx

```diff
@@ -14,45 +16,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [goodVisible, setGoodVisible] = useState(goodsFromServer);
+  const [sortField, setSortField] = useState('');
+
+  const sortAlphabet = () => {
+    if (sortField.includes('reverse')) {
+      setGoodVisible([...goodVisible].sort((a, b) => b.localeCompare(a)));
```

Comment:
It's better to update the state based on the current state using the functional update form of setState. This ensures that you're not mutating the existing state directly and avoids potential side effects
```
setState(prevValue => ...